src/rules/missing-playwright-await.test.ts

```diff
@@ -247,6 +247,21 @@ runRuleTester('missing-playwright-await', rule, {
         },
       },
     },
+    // Page methods
```

Comment:
Would love to hear more ideas on test cases here. It's a bit simpler to check page methods than the test/except cases, since it's not a chained call.

Reply:
Let's add test cases to assert `myPage.goto`, `frame.goto`, `myFrame.goto`, etc.